src/_utils.ts

```diff
@@ -55,6 +56,17 @@ export async function sendFetch (url: string, options?: RequestInit) {
   return await fetch(url, requestOptions);
 }
 
+export function cacheDirectory () {
+  const dotGiget = resolve(homedir(), ".giget");
+  if (platform() === "linux" && !existsSync(dotGiget)) {
+    const CACHE_HOME = process.env.XDG_CACHE_HOME || resolve(homedir(), ".cache");
```

Comment:
I think we shall always respect env regardless of platform (https://github.com/unjs/rc9/blob/14a920f6fae3056208f89d57d60713a9191a2a5c/src/index.ts#L77)